src/code/dialogs/checklist.js

```diff
@@ -297,9 +313,9 @@ const OperationChecklistDialog = WDialog.extend({
 
   getListDialogContent: function (operation, items, sortBy, sortAsc) {
     const content = new Sortable();
-    content.fields = this.getFields(operation);
-    content.sortBy = sortBy;
+    content.sortBy = sortBy; // Need to set sortBy/Asc before fields are set so column header gets sort triangle css class
```

Comment:
the reason sounds wrong, the fix in `sortable` should set the css class whatever the order

Reply:
The fix in sortable is called in getFields and it refers to sortBy and sortAsc to determine which column header gets the css.

Reply:
...so we need to set those before we set the fields.

Reply:
I see you mean that the table header is set on `.fields =`. 
but you can change this so header is updated on `.sortBy/.sortAsc =` instead, maybe moving some logic from `renderHead->...->on click()` to another place in code